includes/size-analysis/section-android.mdx

```diff
@@ -1,10 +1,10 @@
 This guide assumes you already use the Sentry Android Gradle plugin in GitHub Actions. {/* TODO: Link to setup guide */}
```

Comment:
See the [Android Gradle configuration guide](https://docs.sentry.io/platforms/android/configuration/gradle/) for initial setup